gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/hive/HiveUtils.java

```diff
@@ -163,10 +163,4 @@ private static Configuration getHadoopConfiguration() {
     return conf;
   }
 
-  /**
-   * @return true if {@link Table} is partitioned.
-   */
-  public static boolean isPartitioned(Table table) {
```

Comment:
Please leave the method and mark as deprecated. It is possible other products are relying on this method as a library.

Reply:
Sure, added method back with deprecation note. Thanks

Reply:
I don't see the method added back, did you push the change?